views/index.ejs

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <h1>Simple Fucking Link Shortener</h1>
+  <h3>with analytics and shit, too - DAYUM!!!</h3>
+</div>
+
+<div class="container">
+  <form action="/links" method="POST">
+    <div class="form-group">
+      <label for="url">Link to Shorten</label>
+      <input type="text" class="form-control" id="url" name="url">
```

Comment:
res.redirect needs http:// or it wont redirect to the address. By setting the input type to url instead of text it does a front end validation and alerts the user that the address is invalid. type="url"